docs/tutorials/setup-hcloud-cli.md

```diff
@@ -0,0 +1,158 @@
+# Setup the hcloud CLI
+
+This tutorial will guide you through the process of setting up the hcloud CLI on your local machine.
+
+## Prerequisites
+
+Before you begin, ensure you have the following:
+
+- A [Hetzner Cloud account](https://console.hetzner.cloud).
+
+## 1. Install the hcloud CLI
+
+### 1.1 Manual installation
+
+You can download pre-built binaries from our [GitHub releases](https://github.com/hetznercloud/cli/releases). 
+Install them by extracting the archive and moving the binary to a directory in your `PATH`.
```

Comment:
I prefer that user use this method to install the cli. To make this easier and more approachable, could we have 2, 3 command to download and setup the binary on linux?